src/odemis/driver/xt_client.py

```diff
@@ -612,6 +624,144 @@ def set_autostigmator(self, channel_name, state):
             self.server._pyroClaimOwnership()
             return self.server.set_autostigmator(channel_name, state)
 
+    def get_pitch(self):
+        """
+        Get the pitch between two beams within the multiprobe pattern.
```

Comment:
Could be nice to add that it are two directly neighbouring beams (at least that is what I expect).